### ISSUE TITLE
Gtk.init

### ISSUE THREAD

**Issue by shevegen** [mvz/gir_ffi-gtk#19]
Hi.

I see the example:

    require 'gir_ffi-gtk2'
    Gtk.init

Could Gtk.init become optional? In ruby-gnome this already is.

Alternatively, perhaps one could write a init.rb file that does this.

Then we could require it like so:

    require 'gir_ffi-gtk2/init'

Example for the latter also here:

    https://github.com/yuki24/did_you_mean

At extra features

    require 'did_you_mean/extra_features'

Simple but "effective" trick. Anyway if this is not a good fit for the
project, just close the issue request here. Thanks!

**Comment by mvz** [comment]
> Could Gtk.init become optional?

Yes, that makes sense. I believe Gtk has a special version of init that can be called multiple times.
